client/containers/Dashboard/DashboardContent/Settings/Settings.js

```diff
@@ -23,6 +23,25 @@ const propTypes = {
 class Settings extends Component {
 	constructor(props) {
 		super(props);
+
+		/* Export & Delete Mailto Props */
+		this.exportEmailBody = `
```

Comment:
I did some research and I think adding props to the constructor is the best way to do this -- I can't just make it a constant in the file because I need to access props. And of course it shouldn't be state. But if there's a better way, lmk and I'll refactor.